### PR TITLE
Replace deprecated Mobius (C#) with .NET

### PR DESCRIPTION
**Pull request by manojmallela**
Updates `Packages/Language Bindings` with .NET API.

### CODE REVIEW

**Comment by eliasah** [comment]
Hello @manojmallela I'd rather we keep Mobius for now even though it's deprecated. You can also add the .Net connector

**Review by zero323 — CHANGES_REQUESTED** [review]
Could you please make changes, as request here https://github.com/awesome-spark/awesome-spark/pull/182#issuecomment-647954498? Thanks!

**Comment by manojmallela** [comment]
Hi @zero323, refactored as per the comment, please review.

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Users of Apache Spark may choose between different the Python, R, Scala and Java
 
 * [Flambo](https://github.com/yieldbot/flambo) <img src="https://img.shields.io/github/last-commit/yieldbot/flambo.svg"> - Clojure DSL.
 * [Mobius](https://github.com/Microsoft/Mobius) <img src="https://img.shields.io/github/last-commit/Microsoft/Mobius.svg"> - C# bindings.
```
Comment (zero323) [inline README.md:48]: We could add something around these lines:

> (Deprecated in favor of .NET for Apache Spark)

**Review by zero323 — APPROVED** [review]
The comment notwithstanding, LGTM

**Pull request merged**